packaging/build_ffmpeg.sh

```diff
@@ -155,6 +160,13 @@ if [[ "$(uname)" == Darwin ]]; then
         ${prefix}/lib/${swscale}.dylib
     ${otool} -l ${prefix}/lib/${swscale}.dylib | grep -B2 ${prefix}
 
+    ${install_name_tool} \
+        -change ${prefix}/lib/${avutil}.dylib @rpath/${avutil}.dylib \
+        -delete_rpath ${prefix}/lib \
+        -id @rpath/${swresample}.dylib \
+        ${prefix}/lib/${swresample}.dylib
+    ${otool} -l ${prefix}/lib/${swresample}.dylib | grep -B2 ${prefix}
+
```

Comment:
There is only a dependency on avutil:



```
~/dev/torchcodec (add_swresample*) » ldd $CONDA_PREFIX/lib/libswresample.so | grep av                                                                              nicolashug@nicolashug-fedora-PF2MMKSN
	libavutil.so.58 => /home/nicolashug/.miniconda3/envs/codec/lib/./libavutil.so.58 (0x00007f2924800000)
```

I don't have access to a mac so I cannot provide the output of `otool` on the actual dylib as requested on https://github.com/pytorch/torchcodec/pull/219#issuecomment-2369161421, but that should be fine?